Documentation/artifact-format-v3.md

```diff
@@ -572,9 +382,38 @@ This file is complementing the information contained in the header.tar.gz.
 It can have the same structure as header.tar.gz, but for security reasons
 (this file is not signed) only certain files and parameters are allowed.
 
-At the moment ONLY header-info and type-info files are allowed which can
-contain only `artifact_depends` `rootfs_image_checksum`
-parameters and type of the update.
+These files and attributes are allowed:
+
+* `header-info` file with one list attribute:
+  ```
+  {
+    "updates": [
+        {
+            "type": "rootfs-image"
+        },
+        {
+            "type": "delta-image"
+        }
+    ]
+  }
+  ```
+  The `updates` attribute is expected to be in the same order as the original in
+  `header.tar.gz`, and will override it.
+
+* `type-info` file with `artifact_depends` and `rootfs_image_checksum`:
+  ```
+  {
+    "type": "rootfs-image"
+    "artifact_depends": [
+        {
+            "rootfs_image_checksum": "4d480539cdb23a4aee6330ff80673a5af92b7793eb1c57c4694532f96383b619"
+        },
```

Comment:
So it can depend on multiple checksums?

Reply:
Hmm, no, this wasn't the intention. Well spotted.

I think the intention was to simply allow several key/value pairs. But we don't need a list for that. Maybe this one is wrongly defined, and should simply be a map with keys and values. I think this applies to `artifact_provides` as well.

Reply:
:+1:

Reply:
@kacf Also remember to remove the `It can contain zero or more elements`, sentence, along with specifying which parameters are required and not. Maybe:
element (required)
would be a nice syntax?

Reply:
Finally got back to this: Fixed!